docs/docs/integrations/platforms/oci.mdx

```diff
@@ -0,0 +1,32 @@
+# Oracle Cloud Infrastructure (OCI)
+
+The `LangChain` integrations related to the
+[Oracle Cloud Infrastructure](https://www.oracle.com/artificial-intelligence/) platform
+
+## LLMs
+
+### OCI Data Science Model Deployment Endpoint
+
+> [OCI Data Science](https://docs.oracle.com/en-us/iaas/data-science/using/home.htm) is a
+> fully managed and serverless platform for data science teams to build, train, and manage
+> machine learning models in the Oracle Cloud Infrastructure.
+
+The model can be deployed on OCI Data Science and exposed as the OCI Model Deployment Endpoint.
+If you deployed LLM on
+[OCI Data Science Model Deployment Service](https://docs.oracle.com/en-us/iaas/data-science/using/model-dep-about.htm)
+with VLLM or TGI framework, you can use the `OCIModelDeploymentVLLM` or `OCIModelDeploymentTGI`
+classes to interact with it.
```

Comment:
we recommend removing this and follow the content/detail level supplied by other providers. This page is meant to serve as a landing page with links to more detailed information about integrations in the specific docs. Maybe we can change a bit to combine the 1st and 2nd paragraphs: "OCI Data Science is a fully managed and serverless platform for data science teams. Using the OCI Data Science platform you can build, train, manage machine learning models, and then deploy them as OCI Model Deployment Endpoint using the OCI Data Science Model Deployment Service."